doc/lms.mysql

```diff
@@ -1719,6 +1719,7 @@ CREATE TABLE rttickets (
   nodeid int(11) DEFAULT NULL,
   netnodeid int(11) DEFAULT NULL,
   netdevid int(11) DEFAULT NULL,
+  verifier_userid int(11) DEFAULT NULL,
```

Comment:
A dlaczego po prostu nie **verifierid**?

Reply:
Analogia do requestor_userid. Czyli jeśli zmieniać to i tam jak myślisz?

Reply:
Chyba że wzorujemy się na "owner" to powinno być samo "verifier".

Reply:
@chilek jaka decyzja?:)

Reply:
ok, poprawione